services/grizzly-reduce-monitor/src/grizzly_reduce_monitor/common.py

```diff
@@ -104,12 +104,13 @@ def _list_objs(self, endpoint, query=None):
 
             yield from resp_json["results"]
 
-    def list_crashes(self, query=None):
+    def list_crashes(self, query=None, ordering=None):
         """List all CrashEntry objects.
 
         Arguments:
             query (dict or None): The query definition to use.
                                   (see crashmanager.views.json_to_query)
+            ordering (list or None): Field(s) to order by (eg. `id` or `-id`)
```

Comment:
Don't you also want to pass this to the call to _list_objs()?

Reply:
Either way is fine I guess